WebXR API.md

```diff
@@ -70,22 +90,42 @@ A PC with no attached HMD could expose single a flat display.
 
 A script that wishes to make use of an XRDisplay can request an XRSession. This session provides a list of the available realities that the script may request as well as access to the frame of reference,  and sampling frames.
 
-_The XRSession plays the same basic role as the VRSession, with the addition of Reality management._
+The blur event on XRSession, like on VRSession, will be fired when there is no reason to render, for example when the user removes their HMD for a moment but does not terminate the session. Focus will be fired when the user first dons their HMD and then when they return after a blur. Flat screen based displays will fire a focus when they are visible and active and blur when they are not active.
+
+There is a second type of focus and blur on the XRLayer. See below.
+
+_The XRSession plays the same basic role as the VRSession, with the addition of Reality management. The `baseLayer` attribute has been renamed to `layer` because the Reality will now be the rearmost "base" layer with the XRLayer of each session composited on top._
+
+	dictionary XRSessionCreateParametersInit {
+		required boolean exclusive = true;
+	};
+
+	interface VRSessionCreateParameters {
+		readonly attribute boolean exclusive;
+	};
+
+### Todo
+
+- 'exclusive' needs to be rethought given the new use of XRDisplay for magic window. Do we still need sessions that just wants sensor data? 
 
 ## Reality
 
 	interface Reality : EventTarget {
 		readonly atttribute DOMString realityName;
-		readonly attribute XRCoordinates stageLocation;
+		readonly attribute boolean hasStageBounds;
+		readonly attribute XRStageBounds? stageBounds;
```

Comment:
I like that you moved stage bounds out of XRFrameOfReference. It felt strange to have it there there. 

However, I’m not sure that stageBounds should actually be on the Reality interface either. Should the stage bounds change based on the current reality? 

I think the stage bounds are tied to the tracking system. It seems to define the shape of the physical play area (the area in which the user can expect high fidelity tracking), and the reality probably shouldn’t be able to redefine that shape, since the reality has no control over the configuration of the tracking system.